tensorflow/python/platform/flags.py

```diff
@@ -60,6 +63,18 @@ def __setattr__(self, name, value):
     if not self.__dict__['__parsed']:
       self._parse_flags()
     self.__dict__['__flags'][name] = value
+    self._assert_required()
```

Comment:
Only assert the flag it's setting here, not all flags.

There is no need to check again for everything else, it also matches gflags.